recipes/lib-pod5/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: lib-pod5
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/cp311/l/lib_pod5/lib_pod5-{{ version }}-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
Okay I understand it now - this should be pointing to the source code, not the wheel.

Reply:
Thanks, fixed that now - anything else that needs to be fixed?